src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
```

Comment:
It seems that you're extending the 'Figure' class within 'Rectangle'. If 'Figure' is an abstract class designed to set behavior, consider using an interface to define the behavior instead, as abstract classes are generally used for sharing code among closely related classes. If 'Figure' is already an interface, then it's correctly implemented.